test_parser.py

```diff
@@ -65,6 +66,9 @@ def test_parser(zone, data_type, target_datetime):
               'ouput. Full ouput: \n\n{}\n'.format(res))
         return
 
+    assert not any([isinstance(e['datetime'], pd.Timestamp) for e in res_list]), \
```

Comment:
I would do the opposite: assert that all datetimes are instance of `datetime.datetime`.
That way, we also catch datetimes that are strings instead of datetime objects!

Reply:
Makes sense. I wasn't sure if we also accepted arrow instances, but I'll change it. Would be good to keep it consistent anyways.